.github/workflows/pull-request.yaml

```diff
@@ -10,3 +10,7 @@ jobs:
     name: PR
     uses: canonical/observability/.github/workflows/charm-pull-request.yaml@main
     secrets: inherit
+
+  terraform-checks:
+    name: Terraform
+    uses: canonical/observability/.github/workflows/terraform-quality-checks.yaml@OPENG-2685
```

Comment:
Remember to replace it with main at merging (I guess you'll be merging all three PRs at the same time) :)